### PR TITLE
Add world coordinate names to coordinate map aliases

### PR DESCRIPTION
**Pull request by Cadair**
To be honest I really thought this was already the behaviour, so I made it that way.

As well as using `world_axes_names` for aliases in `CoordinatesMap` this actually fixes some tests so they, you know, test things :eyes:

### CODE REVIEW

**Comment by Cadair** [comment]
Hmm, the figure test fail is my fault, but unnecessary.

**Comment by pllim** [comment]
Not sure who besides Tom R is qualified to review this...

**Comment by Cadair** [comment]
@astrofrog plz can haz review? :grinning:

**Comment by astrofrog** [comment]
Yep! The image tests are failing though right?

**Comment by Cadair** [comment]
I am reasonably sure it's supposed to, but I will double check.

File: astropy/visualization/wcsaxes/coordinates_map.py [hunk 1]
```diff
@@ -96,7 +96,10 @@ def __init__(self, axes, transform=None, coord_meta=None,
             # Set up aliases for coordinates
             if isinstance(name, tuple):
                 for nm in name:
-                    self._aliases[nm] = index
+                    # Do not replace an alias already in the map if we have
+                    # more than one alias for this axis.
+                    if nm.lower() not in self._aliases:
```
Comment (astrofrog) [inline astropy/visualization/wcsaxes/coordinates_map.py:101]: remove lower? or check that both the lowercase and original are not in aliases

**Review by astrofrog — COMMENTED** [review]
Looks good but minor question below. Also if image failures are deliberate can you open a PR to add new baseline images?

File: astropy/visualization/wcsaxes/coordinates_map.py (hunk 1 above)
Comment (Cadair) [reply astropy/visualization/wcsaxes/coordinates_map.py:101]: line 102 below is the only place we add things to `self._aliases` and we do it with a lower?

**Comment by Cadair** [comment]
For the record the image change is deliberate and it's to test that when specified the axes are labelled with the world names and not the physical types. This isn't a change introduced in this PR, that is existing behaviour I just noticed it wasn't being tested.

File: astropy/visualization/wcsaxes/coordinates_map.py (hunk 1 above)
Comment (astrofrog) [reply astropy/visualization/wcsaxes/coordinates_map.py:101]: line 102 doesn't show a lower, only line 104 does?

**Comment by astrofrog** [comment]
Ok can you open a PR for the image tests? (to the data repo)

File: astropy/visualization/wcsaxes/coordinates_map.py (hunk 1 above)
Comment (Cadair) [reply astropy/visualization/wcsaxes/coordinates_map.py:101]: that's what I meant :facepalm:

Comment (astrofrog) [reply astropy/visualization/wcsaxes/coordinates_map.py:101]: But my point is that on line 102 we also add things to aliases *without* a ``lower``? Shouldn't both lines 102 and 104 use lower?

**Review by astrofrog — CHANGES_REQUESTED** [review]
One remaining comment below and a comment in https://github.com/astropy/astropy-data/pull/97#pullrequestreview-514163141 - once these are resolved this is good to go.

File: astropy/visualization/wcsaxes/coordinates_map.py (hunk 1 above)
Comment (Cadair) [reply astropy/visualization/wcsaxes/coordinates_map.py]: I just added the lower to all the things?

**Comment by Cadair** [comment]
Looks like I forgot one of the images: https://github.com/astropy/astropy-data/pull/99

**Review by astrofrog — APPROVED** [review]
Looks good, thanks!

**Pull request merged**